packages/labs/src/package.bzl

```diff
@@ -0,0 +1,14 @@
+"Install toolchain dependencies"
+
+load("@build_bazel_rules_nodejs//:defs.bzl", "yarn_install")
+
+def npm_bazel_labs_dependencies():
+    yarn_install(
+        name = "build_bazel_rules_typescript_protobufs_compiletime_deps",
```

Comment:
nit: poorly named as it is now in `build_bazel_rules_nodejs`

Reply:
also, we could probably remove this extra yarn_install entirely by moving its deps to the @bazel/labs package.json but not a big deal as its in labs which is experimental code anyway

Reply:
yeah also I seem to recall this workspace name appeared in a few places which is why we didn't rename it last time we moved it. Going for a minimal edit here to clear out the 1.0 space and yeah, don't mind a mess in labs